.circleci/config.yml

```diff
@@ -1016,12 +1016,15 @@ jobs:
       - store_artifacts: *artifacts_test_results
       - gitter_notify_failure_unless_pr
 
+  # NOTE: work around to git unsafe repository error by adding /root/project as a safe.directory
+  # See https://github.com/CircleCI-Public/cimg-base/issues/170
```

Comment:
Can you put it directly above the workaround? Otherwise it looks as if the whole job was the workaround :P

Reply:
> Have you checked if the resulting `soljson.js` after running `scripts/build_emscripten.sh` and the `soljson.js` CI artifact of the `b_ems` build on the same commit are still binary-identical with these changes?

No, but I just did it now, and they differ :0
```
diff -q soljson.js soljson-ci.js 
Files soljson.js and soljson-ci.js differ
```
Need to investigate where now, because the diff is pretty long.